src/komponenter/htmlPDF.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useContext } from "react";
+import { NavIkon } from "../ikoner/navikon";
+import "../stiler/htmlPDF.css";
+import parse from "html-react-parser";
+import { tabellObjekter } from "../mockdata/tabeller";
+import { SkjemaContext } from "../context/context";
+import {
+	avsnittTilParagraftabell,
+	beregnAntallLinjerIParagraf,
+	MAKS_ANTALL_LINJER,
+	settInnTabell,
+} from "../utils/htmlPdfUtils";
```

Comment:
Disse funksjonene og variabelen brukes kun her. Gir det menig å ha det i en egen fil da?

Reply:
Gjelder ikke for settInnTabell ser jeg 😅